winch/codegen/src/codegen/call.rs

```diff
@@ -0,0 +1,144 @@
+use super::CodeGenContext;
```

Comment:
A top-level module comment describing the calling convention would be really helpful here! In particular, which registers are clobbered or saved, and where arguments are passed.

Reply:
Agreed! Added the documentation in the abi module, and added a reference to it from here.